projects/pillow/fuzz_pillow.py

```diff
@@ -0,0 +1,47 @@
+
+def TestOneInput(data):
+    ImageFile.LOAD_TRUNCATED_IMAGES = True
+    # We need to write a jpeg file first because otherwise libFuzzer will complain that no coverage eixsts
+    # We must trigger execution of the native code even in the first empty try-out runs.
+    if os.path.isfile("/tmp/newfile"):
+        os.remove("/tmp/newfile")
+    with open("/tmp/newfile", "wb+") as rb:
+        rb.write(encoded3[0:1000])
+
+    # Now append our random data
+    with open("/tmp/newfile", "ba+") as rb:
+        rb.write(data)
+
+    # Fuzzer entrypoint
+    with Image.open("/tmp/newfile", "r") as f:
+        f.load()
+    return
+
+def main():
+    atheris.Setup(sys.argv, TestOneInput, enable_python_coverage=False)
+    atheris.Fuzz()
```

Comment:
You might want to add `warnings.filterwarnings("ignore")` since pillow might be ultra-verbose with malformed files.

Reply:
Thanks!